src/frontend/src/repositories/MeterRepository.js

```diff
@@ -14,3 +14,6 @@ export default {
     return Client.put(`${resource}`, meters)
   },
 }
+export function getMeterDetailRoute(id) {
+  return `/meters/${id}`
+}
```

Comment:
To be more consist with the rest of the code, could you add this above as `detail(...)`? Is this maybe already there as `get(meterId)` ?